RegexandParsing/DetectFloatingPointNumbers.py

```diff
@@ -0,0 +1,14 @@
+'''
+Title     : Detect Floating Point Numbers
+Subdomain : Regex and Parsing
+Domain    : Python
+Author    : Atharva Shah
+Created   : 3 April 2021
+Problem   : https://www.hackerrank.com/challenges/introduction-to-regex/problem
+'''
+
+from re import match, compile
+
+pattern = compile('^[-+]?[0-9]*\.[0-9]+$')
```

Comment:
This regex pattern can be changed to `compile('^[-+]?\d*\.\d+$')`

Reply:
Certainly. Done.